app/api/notifications.py

```diff
@@ -0,0 +1,59 @@
+from flask.ext.restplus import Namespace
+
+from app.models.notifications import Notification as NotificationModel
+from app.helpers.data import DataManager
+from app.helpers.data_getter import DataGetter
+from .helpers.helpers import (
+    can_create,
+    requires_auth
+)
+from .helpers.utils import PAGINATED_MODEL, ServiceDAO, \
+     POST_RESPONSES
+from .helpers.utils import Resource
+from .helpers import custom_fields as fields
+
+api = Namespace('notifications', description='Notifications', path='/')
+
+NOTIFICATION = api.model('Notification', {
+    'id': fields.Integer(required=True),
+    'email': fields.String(required=True),
+    'title': fields.String(),
+    'message': fields.String(),
+    'action': fields.String(),
+    'received_at': fields.DateTime(),
+})
+
+NOTIFICATION_PAGINATED = api.clone('NotificationPaginated', PAGINATED_MODEL, {
+    'results': fields.List(fields.Nested(NOTIFICATION))
+})
+
+NOTIFICATION_POST = api.clone('NotificationPost', NOTIFICATION)
+del NOTIFICATION_POST['id']
+
+
+# Create DAO
+class NotificationDAO(ServiceDAO):
+    version_key = 'notifications_ver'
+
+    def create_user_notify(self, payload):
+        user = DataGetter.get_user_by_email(payload['email'])
+        DataManager().create_user_notification(user, payload['action'], payload['title'], payload['message'])
+        return user
```

Comment:
you are returning user here whereas API response is set to NOTIFICATION. So response won't be returned correctly.